### ISSUE TITLE
Chinese language classification error

### ISSUE THREAD

**Issue by welltoo** [Chia-Network/chia-blockchain#2472]
Chinese classification should be Chinese (Taiwan) and Chinese (mainland), rather than the current Chinese and Chinese (China) classification.
Correct classification:中文（中国台湾）和中文（大陆）。

**Comment by maomingyang1314** [comment]
Yes, I also think this is incorrect, I hope developers can consider the vast number of Chinese users.

**Comment by 4jin** [comment]
> 中文分类应为中文（台湾）和中文（大陆），而不是当前的中文和中文（中国）分类。
> 正确分类：中文（中国台湾）和中文（大陆）。

Traditional Chinese
Simplified Chinese
